sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/providers/BigQueryStorageWriteApiSchemaTransformProviderIT.java

```diff
@@ -61,7 +61,7 @@
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
-public class BigQueryStorageWriteApiSchemaTransformProviderTest {
```

Comment:
This breaks DataflowV1 and V2 tests. Any reason moving them from unit test to integration test?

Reply:
CC: @ahmedabu98 @kennknowles @damondouglas

They use fake BigQuery service so has to be executed locally. Either exclude them from Dataflow test suites or move back to unit test?

Reply:
Ahh no reason, bad mistake. I'll open a PR to revert this part

Reply:
Thanks for catching this, pls take a look at #30623